src/app.js

```diff
@@ -20,10 +20,12 @@ const app = express();
 const port = process.env.NODE_LOCAL_PORT || 3000;
 
 app.use(cookieParser(process.env.SECRET_KEY));
-// app.use(encryptCookieNodeMiddleware(process.env.SECRET_KEY));
+app.use(encryptCookieNodeMiddleware(process.env.SECRET_KEY));
 
-const { PUBLIC_ROUTES } = constants;
+const { PUBLIC_AUTH_ROUTES } = constants;
 
+const publicPaths = PUBLIC_AUTH_ROUTES.map(({ path }) => path);
```

Comment:
This _might_ be called `publicAuthPaths`, if we're going to need `publicPaths` for something else. Other than that, it looks pretty good. Thanks for your quick response.